teams.md/docs/main/developer-tools/devtools/cards.md

```diff
@@ -6,10 +6,10 @@ llms: ignore
 
 # 🪪 Cards
 
-![Card Designer Typescript](https://github.com/microsoft/teams.ts/blob/main/assets/screenshots/card_designer_typescript_editor.png?raw=true)
-
 Use the Cards page to design and test your cards. Then, use the "Attach card" button to add that card as an attachment to your message. By default, the card will be attached in the new message compose box, but you can also attach a card when editing an existing message.
 
+![Card Designer Typescript](https://adaptivecards.microsoft.com/designer)
```

Comment:
Sorry to cause confusion, what I meant here was that the adaptive cards designer is iframed in devtools now. I will try to take a screenshot and share here so you can use that in the docs changes :) let me get back to you

Reply:
<img width="1749" height="1262" alt="image" src="https://github.com/user-attachments/assets/8e4ca42f-0c2e-40f6-9009-c89b01bc95c1" />
Please use this screencap saved to the screenshots folder :)